pkgs/top-level/python-packages.nix

```diff
@@ -16788,6 +16788,8 @@ self: super: with self; {
 
   symbolic = callPackage ../development/python-modules/symbolic { };
 
+  symbolic_10 = callPackage ../development/python-modules/symbolic/10.nix { };
```

Comment:
Versioned attributes are not allowed in `python-packages.nix`. Please use `packageOverrides`.

Reply:
Thanks, I didn't know that. I just assumed it was ok since there are some other versioned attributes as well (e.g. `django_{3,4,5}`, `tree-sitter_0_21` and `beancount_2`). Anyway, I just opened #400159